python-sdk/src/astro/databases/databricks/delta.py

```diff
@@ -232,7 +232,7 @@ def table_exists(self, table: BaseTable) -> bool:
             )
             return True
         except ServerOperationError as s:
-            if "Table or view not found" in s.message:
+            if "TABLE_OR_VIEW_NOT_FOUND" in s.message:
```

Comment:
I think this qualifies to be moved to a constant and we re-use the constant in the tests so that we don't need to change this in 2 places next time there is a change, no?

Reply:
@pankajkoti I just noticed that the test should be using this same method - so we have a single occurrence of this string. Do you think it is still worth having a constant for it?

Reply:
No strict opinion. :) Marking the conservation as resolved.